amy/settings.py

```diff
@@ -131,6 +131,7 @@
     'rest_framework',
     'api',
     'captcha',
+    # 'debug_toolbar',
```

Comment:
Well `debug_toolbar` is supposed to run if and only if `DEBUG=True` in the settings. Are you fine with adding it to the requirements and enabling (==uncommenting that line)?